####-release_cycle.md

```diff
@@ -0,0 +1,343 @@
+- The minor release before the next major (aka, *final minor release*) has support for 6 months after the next major. That is, if `X.Y` is the last feature release before `X+1.0`, then `X.Y+1` is the final minor release.
+- The final minor release `X.Y+1` should not have new features but might include `DeprecationWarning`s that will be removed in the next major, `X+1.0`. 
+
+**Patch Version (Z):** Bumps for backward-compatible bug fixes.
+
+- New `Z` releases have no scheduled and they are done on a *need-base* considering:
+  - The *backportability* of the patch
+  - The severity of the bug to fix
+  - The criticality of the affected use-case
+- Cannot be introduced in patch releases:
+  - new deprecation warnings
+  - new features of any kind
+  - raise minimum requirements for using or building Qiskit (including adding new dependencies)
+  - any kind of changes in any kind of public API (including Experimental APIs), including its removal or expansion.
+- It may include:
+  - expansive support for new versions of Python, Rust, or run/build environment
+  - support for new versions of other existing dependencies
+- The support of `X.Y.Z` finishes with the release of `X.Y.Z+1` (or `X.Y+1.0`) in most of the cases (exception might apply)
+- After the release of `X+1`, bugfixing is provided via `X.Y.Z+N` in undetermined regularity for 6 months. 
+
+### Git branching model
+
+Before the release of a major `X.0`, the last minor `X-1.Y` should be tagged in the `stable/X-1.Y`. Two weeks before the release `X.0` the branch `next/X` should be merged into `main` (probably not necessary the first time). Then `stable/X.0` is branched from `main`, as well as `next/X+1`. In `stable/X.0`, `X.0.0rc1` is tagged and release. Two week later, at stable release time, `X.0.0` is tagged on `stable/X.0` and released.
+
+Similarly, two week before the release of `Y` minor, a `stable/X.Y` branch from `main` is created, from where `X.Y.0rc1` is tagged. When stable `X.Y.0` will be released (two weeks later), the label is on `stable/X.Y`.
+
+All changes, including bug fixes, with backward compatibility are PRed with `main` as base. Bugfix PRs that are possible to port to stable should be tagged as `stable-backport-potential` and a bot ports them to `stable/X.Y` (where `Y` is the last minor).
+At undefined time, a new patch release `X.Y.Z` is tagged on `stable/X.Y` and released.
+
+Non-backward compatible changes, including removal of deprecated code, can be PRed at any point against `next/X+1` branch.
+
+Regularly, `main` should be merge into `next/X+1` to keep it up-to-date. The suggestion is for every minor release, at least.
+
+### QPY support
+
+A new major release `X` has support to dump on QPY in the same version than the final minor version `X-1`. This allows `X` to dump QPY that can be loaded with `X-1`. Minor releases can increase the version of QPY but should always support the QPY version in the final minor release of `X-1` until the end-of-life of `X-1`.
```

Comment:
Maybe something like this?: 

```suggestion
The QPY serialization format is backwards compatible so that a new Qiskit can release can always load a QPY
file generated with an earlier release of Qiskit. However, the format isn't forward compatible, so it's not possible
to load QPY files generated with newer version of Qiskit using an older release. To support compatibility across major version releases, the `qiskit.qpy.dump()` function has an optional argument `version` that lets a user specify a back version of the QPY format emitted by the `dump()` function. There are two constants exposed by
the `qpy` module, `qiskit.qpy.QPY_VERSION` which documents the default format version emitted by `dump()` and
also the maximum version that `load()` supports, and `qiskit.qpy.QPY_COMPATIBILITY_VERSION` which documents the minimum version that `dump()` can emit (while `load()`  can load any version <= `QPY_VERSION`).

To facilitate user migration across major version releases the `qiskit.qpy.dump()` function will always support at least one overlapping version between the `X.0.0` and the `X-1.Y.0` release (where Y is the last minor version of 
that series). This will enable saving QPY format files that can be loaded by both major versions from the newer
release. For example, in Qiskit 1.0.0 the `QPY_VERSION` was `11`, and in Qiskit 0.46.0 the `QPY_VERSION` constant was `10`. For qiskit 1.0.0 the minimum version supported by `dump()`'s `version` argument, and exposed by 
`qiskit.qpy.QPY_COMPATIBILITY_VERSION` was 10. This enabled Qiskit 1.0.0 to generate qpy payloads that could
be loaded by Qiskit 0.46.0.
```